run_tests.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+cd ..
+echo 'Running tests...'
+
+EXCLUDE_TESTS=HTTPDataStoreCSVBenchmarks,HTTPDataStoreBenchmarks,MicroBenchmarks,TDBDataStoreBenchmarks,FakeDataStoreBenchmarks,AverageMicroBenchmarks,SqliteDataStoreBenchmarks,DataStoreCSVBenchmarks,TDBDataStoreCSVBenchmarks,AFF4Benchmark
```

Comment:
Yeah that might be a bug we need to check. You can do it in another PR -
lets just get this one in.

Thanks. This is awesome!

On 25 February 2015 at 11:38, Dionysis Zindros <notifications@github.com>
wrote:

> In run_tests.sh
> <https://github.com/google/grr/pull/113#discussion_r25330660>:
>
> > @@ -0,0 +1,8 @@
> > +#!/bin/bash
> > +cd ..
> > +echo 'Running tests...'
> > +
> > +EXCLUDE_TESTS=HTTPDataStoreCSVBenchmarks,HTTPDataStoreBenchmarks,MicroBenchmarks,TDBDataStoreBenchmarks,FakeDataStoreBenchmarks,AverageMicroBenchmarks,SqliteDataStoreBenchmarks,DataStoreCSVBenchmarks,TDBDataStoreCSVBenchmarks,AFF4Benchmark
>
> Yes, I'm excluding them because they are just benchmark tests. They pass
> if included now. I don't know why, but they are not excluded anyway if the
> label specified is "small". I don't know why the label exclusion doesn't
> work for them. Would you like me to investigate further? Perhaps the label
> directive is ignored for some reason.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/google/grr/pull/113/files#r25330660>.
>